gender_novels/novel.py

```diff
@@ -186,6 +187,41 @@ def get_count_of_word(self, word):
                 count += 1
         return count
 
+    def words_associated(self, word):
+        """
+        Returns a dictionary of the words found after given word
+        Note: words always return lowercase
+
+        >>> from gender_novels import novel
+        >>> summary = "She took a lighter out of her purse and handed it over to him."
+        >>> summary += " He lit his cigarette and took a deep drag from it, and then began "
+        >>> summary += "his speech which ended in a proposal. Her tears drowned the ring."
+        >>> summary += " TBH i know nothing about this story."
+        >>> novel_metadata = {'author': 'Hawthorne, Nathaniel', 'title': 'Scarlet Letter',
+        ...                   'corpus_name': 'sample_novels', 'date': 'long long ago',
+        ...                   'filename': None, 'text': summary}
+        >>> scarlett = novel.Novel(novel_metadata)
+        >>> scarlett.words_associated("his")
+        Counter({'cigarette': 1, 'speech': 1})
+
+        :param wlist:
+        :return: dict{ word : [lst_of_following_words]}
+        """
+        word = word.lower()
+        word_count = Counter()
+        check = False
```

Comment:
the check variable is no longer needed.

Reply:
I need it to log when the wanted word shows up so that the next time the for-loop runs I can pick up the following word

Reply:
Ah, you're right!